### ISSUE TITLE
[macOS] Ignore PS button/Guide input when OS-level shortcuts are enabled

### ISSUE THREAD

**Issue by schm1dtmac** [RPCS3/rpcs3#16024]
### Quick summary

macOS's shortcut to open the Launchpad Games folder using a controller's Home/Guide/PS button conflicts with RPCS3's home menu, with both opening simultaneously as a consequence. RPCS3 should detect this, and ignore inputs from said button as a workaround.
### Details

macOS 11 and above offer users with connected game controllers the option to open Launchpad straight to the Games folder by pressing their Home/Guide/PS Button, and to switch apps by holding it down further, with this option being enabled by default. 
However, this conflicts with RPCS3's handling of the same button for use in opening the in-emulator Home menu, leading to both opening at the same time when the button is pressed (but the macOS Launchpad overrides the RPCS3 one and the desktop as a whole so that's what you end up seeing).

The only viable solution is to ignore inputs from this button in RPCS3 whenever the system gesture is enabled, by querying the appropriate system preference.
### System configuration

Hardware: MacBook Air M2 (4P+4E CPU, 8C GPU, 8GB RAM)
OS: macOS 15.0
RPCS3 build: Latest master, compiled using an arm64 build script based on shinra-electric's script, with commit ID c80342e.

**Comment by Megamouse** [comment]
This is not a bug, and it has nothing to do with rpcs3s home menu.
So the title makes no sense here.

**Comment by mavethee** [comment]
If I understand an issue correctly, you can disable that.
Same conflicts with Steam, I've used Terminal command in the past so idk when it became part of GUI but...
`System Settings` -> `Game Controller` -> `Press HOME button to open Launchpad`
**_It's not the bug of the emulator, it reflects PS3 behavior, macOS features are here to blame._**

**Comment by Darkhost1999** [comment]
Inside pads settings of RPCS3 you can right click any box you want to unmap any keybind.
So the option inside RPCS3 that disables the PS Button is to right click the box for the mapped key bind under PS Button. You can optionally remap the keybind to another key if you have another key available to bind.

As for the option to disable launchpad sounds like it's MacOS responsibility to create that option just like how Microsoft handled the option to disable "guide" opens controller bar or game bar and steam handled the option to disable "guide" focuses steam.

**Comment by mavethee** [comment]
As we both said, option is present, author just doesn't want to disable it 🤷🏻‍♂️ 
So Apple did same thing as Microsoft, thats not the case.
But not sure when PPSSPP implemented said features, it still interferes when `Press HOME button to open Launchpad` is enabled on latest commit.

This is titled wrong as kd said.
Not an emulator bug, but more like feature request to support Apple's stuff.

**Comment by schm1dtmac** [comment]
Thought about this some more since I've had to deal with GameController FW and SDL a few times over the past year, even attempting to mitigate this by disabling the system gesture requires use of GameController framework for input (either via SDL with env hint to bypass HIDAPI or otherwise) which isn't a reasonable solution and wouldn't do anything to avoid trouble for the custom DS3/4/5 pad handlers, I'd rather just detect when this macOS option is enabled system-wide and ignore home/PS button inputs emu-side in that case, doesn't help that we need to be careful to only ignore the *physical* home/PS button in case someone remaps it (can be queried via ```defaults read com.apple.GameController bluetoothPrefsMenuLongPressAction```, IDK if this works on Tahoe as that changes button behaviour a bit given Launchpad's removal).

**Comment by Megamouse** [comment]
@schm1dtmac we could also simply close this now since the user now has the option to use a combo by default

**Comment by schm1dtmac** [comment]
> [@schm1dtmac](https://github.com/schm1dtmac) we could also simply close this now since the user now has the option to use a combo by default

Yeah could do so, macOS 27 also lets you opt specific apps and games out of having this gesture trigger entirely rather than needing to disable it system wide so I’m good closing this.
